experimental/FTheoryTools/docs/src/introduction.md

```diff
@@ -50,12 +50,17 @@ We anticipate the following workflow:
 - Singular loci in suitable codimension (e.g., $1$, $2$, and $3$ if the base space has dimension $3$).
 - Fiber diagrams of the resolved fiber over the originally singular loci, including intersections of $\operatorname{U}(1)$-sections.
 - Gauge group.
-- Topological data (e.g., Euler number)
+- Topological data (e.g., Euler number).
 
 Currently, our primary focus is on the Elephant in the room, that is the crepant resolution. While already functional for numerous setups, especially literature models, it is far from complete. At this point, the largest functionality exists for toric cases, with ongoing work to extend those toric resolution techniques to families of spaces and schemes.
 
-We are also actively expanding our database of supported literature models. At this point, our database includes the models discussed in [LS13](@cite). Consequently, we are already covering infinite families of models (e.g., with $SU(k)$ gauge group, where $k \geq 1`).
-Still, the total number of papers in our database is at this point limited to about $5$. This is to be extended a lot.
+We are also actively expanding our database of supported literature models. At this point, amongst others, our database includes models from the following papers:
+- The Tate Form on Steroids: Resolution and Higher Codimension Fibers [LS13](@cite),
+- F-Theory on all Toric Hypersurface Fibrations and its Higgs Branches [KM-POPR15](@cite),
+- Quadrillion F-Theory Compactifications with the Exact Chiral Spectrum of the Standard Model [CHLLT19](@cite).
+
+Consequently, we are already covering infinite families of models (e.g., with $SU(k)$ gauge group, where $k \geq 1$).
+Still, the total number of papers in our database is at this point limited to about $45$. This is to be extended a lot.
```

Comment:
I think this number is not correct (although I did not check it). But I thought we have about 45 different models (including such models that give us an infinite family, say SU(n)), but that the number of papers from where those models originate is more like 10.

Reply:
You're right I misread this originally